examples/toolbox-clock.php

```diff
@@ -33,4 +33,5 @@
 $messages = new MessageBag(Message::ofUser('What date and time is it?'));
 $response = $chain->call($messages);
 
-echo $response.PHP_EOL;
+// Clock tool forced a structured output
+dump(json_decode($response, true));
```

Comment:
![CleanShot 2024-10-04 at 22 35 10@2x](https://github.com/user-attachments/assets/59e428cc-2b16-426a-a455-47d9354cfb32)

This is now working @chr-hertel, but maybe you can help me polish the part in `ChainProcessor`. I am also not that deep in the ChainProcessor thing and I really don't get the difference between response_format and output_structure there. Thanks